src/meta/src/hummock/compaction/picker/intra_compaction_picker.rs

```diff
@@ -272,18 +276,6 @@ impl IntraCompactionPicker {
             assert!(overlap
                 .check_multiple_overlap(&l0.sub_levels[idx].table_infos)
                 .is_empty());
-            let mut target_level_idx = idx;
```

Comment:
Until we're sure this has no impact, keep the comments

Reply:
No. It may cause some bug...
And I think this operation is unsed after https://github.com/risingwavelabs/risingwave/pull/15523

Reply:
Please be more specific :)

> No. It may cause some bug...

What is the bug? Is it the same bug in this issue? I think the intention of the change here is to disallow trivial move for more than one level.

> And I think this operation is unsed after https://github.com/risingwavelabs/risingwave/pull/15523

Why is the operation here relevant to #15523. I though #15523 is just a batching optimization to amortize the cost of ETCD I/O and memory copy.

Reply:
Because if we can batch several task in once transaction, we do not need to move sstable file across several sub levels.
I think this operation may be dangerous